File: examples/integrity/get_proof/get_proof.go
package main

import (
	"log"

	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/bloock/bloock-sdk-go/v2/entity/record"
)

func main() {
	records := []record.Record{ /* some records */ }

	// we give GetProof an array with the hashes of the records
	integrityClient := client.NewIntegrityClient()
	proof, _ := integrityClient.GetProof(records)
	log.Printf("proof: %+v", proof)
}

File: examples/integrity/get_proof/GetProof.java
import com.bloock.sdk.client.IntegrityClient;
import com.bloock.sdk.entity.integrity.Proof;
import com.bloock.sdk.entity.record.Record;
import java.util.List;

public class GetProof {
    public static void main(String[] args) throws Exception {
        List<Record> records = List.of(/* records */);
        IntegrityClient integrityClient = new IntegrityClient();
        Proof proof = integrityClient.getProof(records);
    }
}